portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -13,6 +13,6 @@ public class HelloWorldServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Pikachu");
```

Comment:
Was this for testing purposes or do you want to keep it ?

Reply:
Oh! I guess I don't actually need it anymore, thank you for pointing that out!